.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+name: GH Actions CI
+
+on:
+  # run every Sunday at 3:53 UTC
+  schedule:
+    - cron: "53 3 * * 0"
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+
+concurrency:
+  group: "${{ github.ref }}-${{ github.head_ref }}"
+  cancel-in-progress: true
+
+jobs:
+  unittests:
+    name: Test ${{ matrix.os }} with Python ${{ matrix.python-version }}
+    if: github.repository == 'MDAnalysis/MDAnalysisData'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        python-version: [2.7, 3.6, 3.7, 3.8, 3.9]
+        exclude:
+          - os: macOS-latest
```

Comment:
the exclude here is weird, I guess you're only testing 3.8 and 2.7 on macOS and Windows? It might be a bit simpler to just not have these in the main matrix and just add them as includes (4 entries instead of 6 in exclude)

Reply:
good call, changing it